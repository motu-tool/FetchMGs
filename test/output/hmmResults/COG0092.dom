#                                                                            --- full sequence --- -------------- this domain -------------   hmm coord   ali coord   env coord
# target name        accession   tlen query name           accession   qlen   E-value  score  bias   #  of  c-Evalue  i-Evalue  score  bias  from    to  from    to  from    to  acc description of target
#------------------- ---------- ----- -------------------- ---------- ----- --------- ------ ----- --- --- --------- --------- ------ ----- ----- ----- ----- ----- ----- ----- ---- ---------------------
1089456.NCGM2_5502   -            228 COG0092              -            362  4.8e-121  404.9   8.0   1   1  1.1e-124  5.2e-121  404.8   8.0     1   221     1   221     1   228 0.94 protein_id="BAK92323.1" gene="rpsC" product="30S ribosomal protein S3" GI="348036963" note="similar to GI:116052292; similar to PA14_08910 of P. aeruginosa UCBPP-PA14" project_id="73453"
1037409.BJ6T_43930   -            241 COG0092              -            362  1.9e-120  403.0   7.9   1   1  4.4e-124  2.1e-120  402.8   7.9     1   234     1   236     1   241 0.91 protein_id="BAL09662.1" gene="rpsC" product="30S ribosomal protein S3" GI="354956983" project_id="67463"
944546.ABED_0705     -            234 COG0092              -            362  7.8e-113  377.9  10.5   1   1  1.9e-116  8.6e-113  377.8  10.5     1   227     1   229     1   234 0.92 protein_id="BAK70422.1" product="30S ribosomal protein S3" GI="345468971" project_id="61719"
944547.ABLL_0980     -            234 COG0092              -            362  8.8e-113  377.7  10.7   1   1  2.1e-116  9.7e-113  377.6  10.7     1   226     1   228     1   234 0.92 protein_id="BAK72855.1" product="30S ribosomal protein S3" GI="345471405" project_id="61721"
1148.SYNGTS_0769     -            240 COG0092              -            362    1e-109  367.6   5.3   1   1  2.4e-113  1.1e-109  367.5   5.3     1   231     1   234     1   240 0.93 protein_id="BAK49517.1" gene="rps3" product="30S ribosomal protein S3" GI="339273030" note="ORF_ID:sll1804" project_id="67081"
#
# Program:         hmmsearch
# Version:         3.3.2 (Nov 2020)
# Pipeline mode:   SEARCH
# Query file:      /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0092.hmm
# Target file:     ../fetchMGs/example_datasets/example_data_genomes.faa
# Option settings: /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/bin/hmmsearch -o output/hmmResults/COG0092.out --domtblout output/hmmResults/COG0092.dom --noali --notextw -T 120 --cpu 1 /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0092.hmm ../fetchMGs/example_datasets/example_data_genomes.faa 
# Current dir:     /nfs/nas22/fs2202/biol_micro_bioinf/software/fetchmgs_dev
# Date:            Wed Sep 14 14:19:37 2022
# [ok]
