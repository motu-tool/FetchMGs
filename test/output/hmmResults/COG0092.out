# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.3.2 (Nov 2020); http://hmmer.org/
# Copyright (C) 2020 Howard Hughes Medical Institute.
# Freely distributed under the BSD open source license.
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0092.hmm
# target sequence database:        ../fetchMGs/example_datasets/example_data_genomes.faa
# output directed to file:         output/hmmResults/COG0092.out
# per-dom hits tabular output:     output/hmmResults/COG0092.dom
# show alignments in output:       no
# max ASCII text line length:      unlimited
# sequence reporting threshold:    score >= 120
# number of worker threads:        1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       COG0092  [M=362]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence           Description
    ------- ------ -----    ------- ------ -----   ---- --  --------           -----------
   4.8e-121  404.9   8.0   5.2e-121  404.8   8.0    1.0  1  1089456.NCGM2_5502  protein_id="BAK92323.1" gene="rpsC" product="30S ribosomal protein S3" GI="348036963" note="similar to GI:116052292; similar to PA14_08910 of P. aeruginosa UCBPP-PA14" project_id="73453"
   1.9e-120  403.0   7.9   2.1e-120  402.8   7.9    1.0  1  1037409.BJ6T_43930  protein_id="BAL09662.1" gene="rpsC" product="30S ribosomal protein S3" GI="354956983" project_id="67463"
   7.8e-113  377.9  10.5   8.6e-113  377.8  10.5    1.0  1  944546.ABED_0705    protein_id="BAK70422.1" product="30S ribosomal protein S3" GI="345468971" project_id="61719"
   8.8e-113  377.7  10.7   9.7e-113  377.6  10.7    1.0  1  944547.ABLL_0980    protein_id="BAK72855.1" product="30S ribosomal protein S3" GI="345471405" project_id="61721"
     1e-109  367.6   5.3   1.1e-109  367.5   5.3    1.0  1  1148.SYNGTS_0769    protein_id="BAK49517.1" gene="rps3" product="30S ribosomal protein S3" GI="339273030" note="ORF_ID:sll1804" project_id="67081"


Domain annotation for each sequence:
>> 1089456.NCGM2_5502  protein_id="BAK92323.1" gene="rpsC" product="30S ribosomal protein S3" GI="348036963" note="similar to GI:116052292; similar to PA14_08910 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  404.8   8.0  1.1e-124  5.2e-121       1     221 [.       1     221 [.       1     228 [] 0.94

>> 1037409.BJ6T_43930  protein_id="BAL09662.1" gene="rpsC" product="30S ribosomal protein S3" GI="354956983" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  402.8   7.9  4.4e-124  2.1e-120       1     234 [.       1     236 [.       1     241 [] 0.91

>> 944546.ABED_0705  protein_id="BAK70422.1" product="30S ribosomal protein S3" GI="345468971" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  377.8  10.5  1.9e-116  8.6e-113       1     227 [.       1     229 [.       1     234 [] 0.92

>> 944547.ABLL_0980  protein_id="BAK72855.1" product="30S ribosomal protein S3" GI="345471405" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  377.6  10.7  2.1e-116  9.7e-113       1     226 [.       1     228 [.       1     234 [] 0.92

>> 1148.SYNGTS_0769  protein_id="BAK49517.1" gene="rps3" product="30S ribosomal protein S3" GI="339273030" note="ORF_ID:sll1804" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  367.5   5.3  2.4e-113  1.1e-109       1     231 [.       1     234 [.       1     240 [] 0.93



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (362 nodes)
Target sequences:                      23272  (7230323 residues searched)
Passed MSV filter:                      1130  (0.0485562); expected 465.4 (0.02)
Passed bias filter:                      681  (0.0292626); expected 465.4 (0.02)
Passed Vit filter:                        74  (0.00317979); expected 23.3 (0.001)
Passed Fwd filter:                         8  (0.000343761); expected 0.2 (1e-05)
Initial search space (Z):              23272  [actual number of targets]
Domain search space  (domZ):               5  [number of targets reported over threshold]
# CPU time: 0.22u 0.01s 00:00:00.23 Elapsed: 00:00:00.17
# Mc/sec: 14885.70
//
[ok]
